components/google_sheets/actions/add-single-row/add-single-row.mjs

```diff
@@ -117,8 +139,8 @@ export default {
       ],
     });
 
-    $.export("$summary", `Successfully added 1 row to [${data.updatedRange} in Google Sheets](https://docs.google.com/spreadsheets/d/${data.spreadsheetId})`)
-  
-    return data
+    $.export("$summary", `Successfully added 1 row to [${data.updatedRange} in [Google Sheets](https://docs.google.com/spreadsheets/d/${data.spreadsheetId})`);
```

Comment:
It looks like the brackets are intended to surround `${data.updatedRange} in Google Sheets`, using markdown syntax.

```suggestion
    $.export("$summary", `Successfully added 1 row to [${data.updatedRange} in Google Sheets](https://docs.google.com/spreadsheets/d/${data.spreadsheetId})`);
```